docs/site/src/content/docs/plugins/language/metrics-output.mdx

```diff
@@ -0,0 +1,129 @@
+---
+title: Metrics Output Plugin
+---
+
+import { PackageManagers } from "starlight-package-managers";
+
+Generate a JSON file of metrics using `@player-tools/metrics-output-plugin`. The plugin consumes diagnostics from LSP plugins and produces a single JSON file with per-file metrics and optional project-level metadata.
```

Comment:
What does "LSP" stand for here?

Reply:
Linking out seems excessive to me. If someone is reading about LSP/language plugins its not an unreasonable assumption that they are aware of what an LSP is. At the minimum though, I think it would be fine to our internal page on how our LSP is architected.

Reply:
For now, I think since both of these plugins are under the `language` section, that it is a little self explanatory. That being said, I think having a more robust Plugin Overview page with glossary terms and additional context would be good.